.eslintrc.js

```diff
@@ -45,6 +45,35 @@ module.exports = {
       extends: ['plugin:node/recommended'],
     },
 
+    // ts files
+    {
```

Comment:
Can remove this if you don't like UBER-STRICTNESS, but I find it useful during dev at least